src/matcha_ml/templates/azure_template.py

```diff
@@ -17,7 +17,7 @@
     "zen_server/zenml_helm/templates",
     "data_version_control_storage",
 ]
-LLM_STACK = DEFAULT_STACK + [
+LLM_STACK_TF = DEFAULT_STACK_TF + [
     "chroma",
     "chroma/chroma_helm",
     "chroma/chroma_helm/templates",
```

Comment:
FYI: `LLM_STACK_TF` and `DEFAULT_STACK_TF` will become redundant with the modular implementation. I have created tickets for their removal and updating wherever they are used 👍